packages/angular_devkit/architect/src/schedule-by-name.ts

```diff
@@ -101,29 +98,26 @@ export async function scheduleByName(
 
   // Start the builder.
   output.pipe(first()).subscribe({
-    error() {},
+    error: () => {},
   });
 
   return {
     id,
     info,
     // This is a getter so that it always returns the next output, and not the same one.
     get result() {
-      return output.pipe(first()).toPromise();
+      return firstValueFrom(output);
+    },
+    get lastOutput() {
+      return lastValueFrom(output);
     },
     output,
     progress: job
       .getChannel<BuilderProgressReport>('progress', progressSchema)
       .pipe(shareReplay(1)),
-    stop() {
+    async stop() {
       job.stop();
-
-      return job.outboundBus
-        .pipe(
-          ignoreElements(),
-          catchError(() => EMPTY),
-        )
-        .toPromise();
+      outboundBusSub.unsubscribe();
```

Comment:
Does this result in any different behavior?
Looks like it was previously waiting for the outbound bus to be complete before resolving.

Reply:
I did not see any different behaviour. But, I can revert this.